docs/.vuepress/sidebar-menus/user-guide.js

```diff
@@ -85,6 +85,18 @@ module.exports = [{
             '/manual/log-filters/multi-line-regex.md'
           ]
         },
+//        {
```

Comment:
Why did we kill this section of navigation?  This may impact Algolia's ability to crawl all the pages linked on these pages.

Reply:
ok.  Is this ready for merge then and the nav will follow later?

Reply:
yup!